llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/TiledOpInterface.h

```diff
@@ -15,17 +15,19 @@
 #include "mlir/Support/LLVM.h"
 
 /// Include the ODS generated interface header files.
-#include "iree/compiler/Dialect/LinalgExt/IR/TiledOpInterface.h.inc"
+#include "iree-dialects/Dialect/LinalgExt/IR/TiledOpInterface.h.inc"
 
 namespace mlir {
 namespace iree_compiler {
-namespace linalg_ext {
+namespace IREE {
```

Comment:
like the namespace changes; being able to use IREE:: as the universal autocomplete root is nice!

Reply:
Yeah, that took a lot of time to clean up and make consistent but helps a lot, imo.